portality/view/api_v1.py

```diff
@@ -40,61 +40,322 @@ def list_operations():
+                            type:
+                              type: string
+                            id:
+                              type: string
+                      subject:
+                        type: array
+                        items:
+                          type: object
+                          title: subject
+                          properties:
+                            code:
+                              type: string
+                            terms:
+                              type: string
+                            scheme:
+                              type: string
+                  created_date:
+                    type: string
+                    format: dateTime
+            query:
+              type: string
+            total:
+              type: integer
+            page:
+              type: integer
+      400:
+        description: Bad Request
+    """
+    # get the values for the 2 other bits of search info: the page number and the page size
+    page = request.values.get("page", 1)
+    psize = request.values.get("pageSize", 10)
+
+    # check the page is an integer
+    try:
+        page = int(page)
+    except:
+        return _bad_request("Page number was not an integer")
+
+    # check the page size is an integer
+    try:
+        psize = int(psize)
+    except:
+        return _bad_request("Page size was not an integer")
+
+    results = None
+    try:
+        results = DiscoveryApi.search_journals(search_query, page, psize)
+    except DiscoveryException as e:
+        return _bad_request(e.message)
+
+    return jsonify_models(results)
+
+@blueprint.route('/search/articles/<path:search_query>')
+def search_articles(search_query):
+    """
+    Search for articles
+    ---
+    tags:
+      - search
+    notes: This is a longer string of important things
+    parameters:
+      -
+        name: search_query
+        description: What you are searching for, e.g. computers
+        in: path
         required: true
-        type: "string"
+        type: string
       -
-        name: "page"
-        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In a augue nulla. In vehicula est eget erat iaculis, et lacinia enim faucibus. Aliquam diam dolor, euismod ultrices viverra vitae, congue sit amet massa. Duis vestibulum, quam ut ultrices sagittis, nunc diam cursus odio, sed efficitur eros erat a sapien."
-        in: "body"
+        name: page
+        description: Which page of the results you wish to see. This field is optional.
+        in: path
         required: false
-        type: "int"
+        type: integer
       -
-        name: "pageSize"
-        description: "Wooo, we can write small essays in here as well. Again limited as it is a string, but pretty neat"
-        in: "body"
-        required: false,
-        type: "int"
+        name: pageSize
+        description: How many results per page you wish to see, the default is 100.
```

Comment:
same comments as for the journal route at same place, maybe have a DEFAULT_ARTICLE_PAGE_SIZE and DEFAULT_JOURNAL_PAGE_SIZE if they're different.